test/cases/x86_64-linux/hello_world_with_updates.0.zig

```diff
@@ -2,4 +2,7 @@
 // output_mode=Exe
 // target=x86_64-linux
 //
-// :110:9: error: root struct of file 'tmp' has no member named 'main'
+// :604:45: error: root struct of file 'tmp' has no member named 'main'
+// :553:12: note: called from here
+// :503:36: note: called from here
+// :466:17: note: called from here
```

Comment:
```suggestion
// :?:?: error: root struct of file 'tmp' has no member named 'main'
// :?:?: note: called from here
// :?:?: note: called from here
// :?:?: note: called from here
```
And same with the other.